src/makeRobot.js

```diff
@@ -39,6 +39,86 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return 'name: '
+      + this.name
+      + ', chip version: '
+      + this.version
+      + ', wheels: '
+      + this.wheels;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(times = 0) {
+      if (times < 0) {
+        return this;
+      }
+
+      if (times > 0) {
+        this.coords.y += 1 * times;
+      } else {
+        this.coords.y += 1 + times;
+      }
+
+      return this;
+    },
+    goBack(times = 0) {
+      if (times < 0) {
+        return this;
+      }
+
+      if (times > 0) {
+        this.coords.y -= 1 * times;
+      } else {
+        this.coords.y -= 1 + times;
+      }
+
+      return this;
+    },
+    goRight(times = 0) {
```

Comment:
```suggestion
    },
    
    goRight(times = 0) {
```
Add newlines between methods to make your code more readable